Extension/src/packageManager.ts

```diff
@@ -150,7 +157,12 @@ export class PackageManager {
             .then((list) => {
                 return list.filter((value, index, array) => {
                     return (!value.architectures || value.architectures.indexOf(this.platformInfo.architecture) !== -1) &&
-                        (!value.platforms || value.platforms.indexOf(this.platformInfo.platform) !== -1);
+                        (!value.platforms || value.platforms.indexOf(this.platformInfo.platform) !== -1) &&
+                        (!this.platformInfo.version || !value.versionRegex || (
```

Comment:
I think this statement isn't right. Right now it is treating a package with a versionRegex and a match that is required but no platform version to allow that download so for some reasonif you weren't able to get a platform.version you end up with both packages?